### PR TITLE
Add ability to change all handlers to use either Vim or IDE shortcuts.

### PR DESCRIPTION
**Pull request by LazyScaper**
Adds the ability to change all handlers to use either Vim or IDE shortcuts.

This is particularly useful in a new instance where you do not use a .ideavimrc file.

### CODE REVIEW

File: src/main/java/com/maddyhome/idea/vim/ui/VimEmulationConfigurable.kt [hunk 1]
```diff
@@ -328,6 +329,38 @@ internal class VimEmulationConfigurable : Configurable {
...
+  ) : DumbAwareAction(
+    "Toggle all Handlers",
+    "Toggle all action handlers to use either IDE or Vim shortcuts",
+    AllIcons.Actions.ChangeView,
+  ) {
+    override fun update(e: AnActionEvent) {
+      e.presentation.isEnabled = true
+    }
+
+    override fun getActionUpdateThread(): ActionUpdateThread = ActionUpdateThread.EDT
```
Comment (AlexPl292) [inline src/main/java/com/maddyhome/idea/vim/ui/VimEmulationConfigurable.kt:344]: Let's switch to ActionUpdateThread.BGT

**Review by AlexPl292 — CHANGES_REQUESTED** [review]
Hi there, that's a cool feature, I like it.

I think it is worth updating it a bit:

- Let's not only switch between "Vim" and "IDE" handlers, but rotate between "Undefined" -> "Vim" -> "IDE" -> "Undefined". I think this will be helpful in case the user clicks the button out of interest and gets scared that they "broke" everything.

- Also, we should probably update not all handlers, but only the ones that are set to "Undefined". 

What do you think about that?

File: src/main/java/com/maddyhome/idea/vim/ui/VimEmulationConfigurable.kt (hunk 1 above)
Comment (LazyScaper) [reply src/main/java/com/maddyhome/idea/vim/ui/VimEmulationConfigurable.kt:344]: Thanks for the feedback, will do another passover tonight with the above in mind.

I like the idea of not updating all handlers, seems "safer" for the user.

Comment (LazyScaper) [reply src/main/java/com/maddyhome/idea/vim/ui/VimEmulationConfigurable.kt]: I have added the above funtionality.

**Comment by AlexPl292** [comment]
Awesome, thank you for the feature!

**Pull request merged**